pkg/rekor/identity.go

```diff
@@ -335,3 +353,65 @@ func oidMatchesPolicy(cert *x509.Certificate, oid asn1.ObjectIdentifier, extensi
 
 	return false, nil, "", nil
 }
+
+func LogIdentitySearch(config LogIdentitySearchConfiguration, rekorClient *gclient.Rekor) error {
+	if (config.StartIndex == nil && config.EndIndex != nil) || (config.StartIndex != nil && config.EndIndex == nil) {
```

Comment:
Can we use previous checkpoint if only start is not given? Can we use current checkpoint if end is not given?

If we can, it would simplify this, as we would have a single method to convert from start index pointer to the value given by user or the last checkpoint and similarly for the end index (with current checkpoint)